skimage/transform/tests/test_geometric.py

```diff
@@ -544,6 +544,39 @@ def test_union_differing_types():
         tform1.__add__(tform2)
 
 
+@pytest.mark.parametrize(
+    "tform",
+    [
+        ProjectiveTransform(matrix=np.random.rand(3, 3)),
+        AffineTransform(scale=(0.1, 0.1), rotation=0.3),
+        EuclideanTransform(rotation=0.9, translation=(5, 5)),
+        SimilarityTransform(scale=0.1, rotation=0.9),
+        EssentialMatrixTransform(
+            rotation=np.eye(3), translation=(1 / np.sqrt(2), 1 / np.sqrt(2), 0)
+        ),
+        FundamentalMatrixTransform(
+            matrix=EssentialMatrixTransform(
+                rotation=np.eye(3), translation=(1 / np.sqrt(2), 1 / np.sqrt(2), 0)
+            ).params
+        ),
+        ((t := PiecewiseAffineTransform()).estimate(SRC, DST) and t),
+    ],
+)
+def test_inverse(tform):
+    assert isinstance(tform.inverse, type(tform))
+    assert_almost_equal(tform.inverse.inverse(SRC), tform(SRC))
```

Comment:
I actually had it more explicit like this, but it only works for ProjectiveTransform since the others have other attribute names. I resorted to the more generic way of applying the transform to points.

I could add a conditional check for `params`, or split the test into one transforms that support `params` and one for the rest.

Reply:
Huh, I tested this locally. The only one missing the `params` attribute is the `PiecewiseAffineTransform`? Though for the test's to pass the more relaxed `assert_almost_equal` is required rather than `assert_array_equal`. Maybe something like this
```suggestion
    try:
        assert_almost_equal(tform.inverse.inverse.params, tform.params)
    except AttributeError:
        assert isinstance(tform, PiecewiseAffineTransform)
    assert_almost_equal(tform.inverse.inverse(SRC), tform(SRC))
```
Though not really an improvement in readability. I'll leave it up to you if and how you want to include it. :)

Reply:
I added the suggested code.